CHANGELOG.md

```diff
@@ -10,6 +10,13 @@
 * [FEATURE] Ingester: Added `-blocks-storage.tsdb.head-chunks-write-queue-size` allowing to configure the size of the in-memory queue used before flushing chunks to the disk . #5000
 * [FEATURE] Query Frontend: Log query params in query frontend even if error happens. #5005
 * [BUGFIX] Updated `golang.org/x/net` dependency to fix CVE-2022-27664. #5008
+[ENHANCEMENT] Distributor: added per-tenant request limit: #5024
+The following CLI flags have been added:
+  - distributor.request-rate-limit
+  - distributor.request-burst-limit
+The following metric is exposed to tell how many requests have been rejected:
+  - cortex_discarded_requests_total
+And change `ingestion_rate_strategy`  to `rate_strategy`
```

Comment:
it should be mentioned that -distributor.ingestion-rate-limit-strategy has been renamed.

Reply:
SGTM.